public/app/features/admin/UserListAdminPage.tsx

```diff
@@ -79,13 +89,16 @@ const UserListAdminPageUnConnected: React.FC<Props> = ({
             />
             <RadioButtonGroup
               options={[
-                { label: 'All users', value: 'all' },
-                { label: 'Active last 30 days', value: 'activeLast30Days' },
+                { label: 'All users', value: false },
+                { label: 'Active last 30 days', value: true },
               ]}
-              onChange={changeFilter}
-              value={filter}
+              onChange={(value) => changeFilter({ name: 'activeLast30Days', value })}
```

Comment:
The { name, value } isn't a SelectableValue so not sure why we're introducing another concept here?